frameworks/hdfs/integration/tests/test_sanity.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import shakedown
+import inspect
+import os
+
+from tests.test_utils import (
```

Comment:
For whatever reason all paths are relative to integration/ rather than integration/tests. For reference scheduler tests just now, I tried dropping `tests.` and found that python couldn't find the module anymore.